### PR TITLE
dev-libs/libpeas: Version bump to 2.0.3

### PR DESCRIPTION
**Pull request by flint2**

### CODE REVIEW

File: dev-libs/libpeas/metadata.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <name>Gentoo GNOME Desktop</name>
   </maintainer>
   <use>
+    <flag name="gjs">Enable GJS support using <pkg>dev-libs/gjs</pkg></flag>
```
Comment (pacho2) [inline dev-libs/libpeas/metadata.xml:9]: I think we could use the global "javascript" name. It seems upstream pretends to support javascript (via GJS)
https://gitlab.gnome.org/GNOME/libpeas/-/issues/47

File: dev-libs/libpeas/libpeas-2.0.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+IUSE="gjs glade +gtk gtk-doc lua +python vala"
+REQUIRED_USE="lua? ( ${LUA_REQUIRED_USE} )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	>=dev-libs/glib-2.44:2
+	>=dev-libs/gobject-introspection-1.39:=
+	gjs? (
+		>=dev-libs/gjs-1.77.1
+		>=dev-lang/spidermonkey-115
```
Comment (pacho2) [inline dev-libs/libpeas/libpeas-2.0.3.ebuild:27]: This should be dev-lang/spidermonkey:115 slot

And probably gjs be raised to 1.77.2 as suggested in https://github.com/gentoo/gentoo/pull/32202#discussion_r1314967052

File: dev-libs/libpeas/metadata.xml (hunk 1 above)
Comment (flint2) [reply dev-libs/libpeas/metadata.xml]: Done in https://github.com/gentoo/gentoo/pull/34835/commits/6cddcffa77b0823b7a569da486e87228203288b7

File: dev-libs/libpeas/libpeas-2.0.3.ebuild (hunk 2 above)
Comment (flint2) [reply dev-libs/libpeas/libpeas-2.0.3.ebuild]: Done in https://github.com/gentoo/gentoo/pull/34835/commits/6cddcffa77b0823b7a569da486e87228203288b7

**Pull request closed without merging**